src/ophyd_async/planstubs/prepare_trigger_and_dets.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+import bluesky.plan_stubs as bps
+
+from ophyd_async.core.detector import DetectorTrigger, StandardDetector, TriggerInfo
+from ophyd_async.core.flyer import HardwareTriggeredFlyable
+from ophyd_async.core.utils import in_micros
+from ophyd_async.panda.table import SeqTable, SeqTableRow, seq_table_from_rows
+from ophyd_async.panda.trigger import SequenceTableInfo
+
+
+def prepare_static_seq_table_flyer_and_detectors_with_same_trigger(
+    flyer: HardwareTriggeredFlyable[SeqTable],
```

Comment:
This type hint seems wrong, shouldn't it be
```suggestion
    flyer: HardwareTriggeredFlyable[SequenceTableInfo],
```
Can you find out why type checking didn't catch this?

Reply:
Yes it should, also can you think of a better name for `SequenceTableInfo`?

Reply:
Also It didn't catch it because it's not technically wrong here, `HardwareTriggeredFlyable[T]` is generic on T, so its got no reason to know it shouldn't be that.

Reply:
I've corrected it and for now changed the name to `SeqTableInfo`.

Reply:
> Yes it should, also can you think of a better name for `SequenceTableInfo`?

`SeqTableAndFriends`? Naming is hard... Maybe `RepeatedSeqTable` is ok?

Reply:
I can make up a test script tomorrow and check it out